vhdbuilder/packer/install-dependencies.sh

```diff
@@ -348,6 +348,11 @@ if grep -q "fullgpu" <<< "$FEATURE_FLAGS" && grep -q "gpudaemon" <<< "$FEATURE_F
 fi
 fi
 
+systemctlEnableAndStart cgroup-memory-telemetry.timer || exit 1
+systemctlEnableAndStart cgroup-memory-telemetry.service || exit 1
+systemctlEnableAndStart cgroup-pressure-telemetry.timer || exit 1
+systemctlEnableAndStart cgroup-pressure-telemetry.service || exit 1
+
```

Comment:
suggest a demo 

```
cat /var/log/azure/Microsoft.Azure.Extensions.CustomScript/events/*
rm -r /var/log/azure/Microsoft.Azure.Extensions.CustomScript || exit 1
````

to validate the script logic, and remove the dir for waagent to own it